### ISSUE TITLE
API link doesn't work

### ISSUE THREAD

**Issue by ACPK** [hmartch/poc_react#2]
Is the link private [https://github.com/MobileXLabs/browser_api](https://github.com/MobileXLabs/browser_api)?

**Comment by hmartch** [comment]
Yes, the browser_api is a private repo of my company.  Also, the poc_react
repo is my first attempt at working with react, and is very much a work in
progress where I am trying to cobble together various things that I am
trying to learn and understand.  I would not recommend using it as the
basis of any learning you might be doing yourself.

Perhaps I should add some sort of disclaimer to the README.  It didn't
really occur to me that someone else might come to it as you have done.

**Comment by ACPK** [comment]
@hmartch - Any suggestions for redirecting users that are logged in with j-toker?  [https://github.com/pducks32/results22](https://github.com/pducks32/results22) has a really weird redirect that shows you the dashboard before sending you to the login screen?

**Comment by hmartch** [comment]
I took at quick look at the pducks32/results22 repo you referenced, which I am not familiar with, and I see that they use the react-routing npm module which I have not used.  You might want to take a look at https://github.com/rackt/react-router which is what I am currently using for routing.

**Comment by ACPK** [comment]
@hmartch - Thanks. I noticed the same bug in the j-token demo where it does a really quick refresh when you load the page.
